src/lib/intents.js

```diff
@@ -1,5 +1,21 @@
 import { cozyFetchJSON } from './stack'
 
+
+// This is a function that does the bare minimum in order to bypass the normal intent flow. To be replaced in th next version of intents.
+export function rawIntent (action, type, data = {}, permissions = []) {
```

Comment:
What the purpose of the rawIntent ? Is it what we called redirection in [this issue](https://github.com/cozy/cozy-client-js/issues/201) ?

Reply:
Wow, I almost missed this one, sorry! It's actually to accomplish what we called complex communication. So if @CPatchane wants something similar for Claudy, here's an ugly way to do it!
…though we'd better update the spec and the API, really.